2_serving_natively_with_amazon_sagemaker/deploy.ipynb

```diff
@@ -59,8 +61,8 @@
     "model = PyTorchModel(model_data=model_artifact,\n",
     "                   name=name_from_base('roberta-model'),\n",
     "                   role=role, \n",
-    "                   # is it worth having a cell where we 'cat' this file and explain the programming interface?
-    "                   # that way, customers can modify it as needed based on the model they want to use.
```

Comment:
My thoughts are that we just want to focus on the serving aspect and highlight that it is TorchServe under the hood. I did however add the following comment to the cell "# Note: You can update the 'torchserve-predictor.py' file as needed according to the model you want to use (ie BERT)"